coriolis_openstack_utils/actions.py

```diff
@@ -1090,3 +1091,143 @@ def execute_operations(self):
+            if networks.check_network_similarity(
+                    src_network, dest_network,
+                    self._source_client, self._destination_client):
+                LOG.info("Found destination network '%s' with same "
+                         "information and subnets as source network '%s'."
+                         % (dest_network_name, src_network['name']))
+                return {"done": True, "result": dest_network['id']}
+
+        if len(conflicting) == 1:
+            raise Exception("Found destination network with "
+                            "with name '%s' but different attributes!"
+                            % dest_network_name)
+        elif conflicting:
+            raise Exception("Found multiple destination networks with "
+                            "with name '%s'! Aborting subnet "
+                            "migration." % dest_network_name)
+
+        return {"done": False, "result": None}
+
+    def equivalent_to(self, other_action):
+        if other_action.action_type == self.action_type:
+            if self.payload['src_network_id'] == (
+                    other_action.payload.get('src_network_id')):
+                return (self.payload.get('dest_tenant_id') ==
+                        other_action.payload.get('dest_tenant_id'))
+        return False
+
+    def print_operations(self):
+        super(NetworkCreationAction, self).print_operations()
+        network_name = self.get_new_network_name()
+        LOG.info(
+            "Create new destination network named '%s'." % network_name)
+
+    def get_source_network_name(self):
+        return networks.get_network(
+            self._source_client, self.payload['src_network_id'])['name']
+
+    def get_new_network_name(self):
+        return self.network_name_format % {
+            "original": self.get_source_network_name()}
+
+    def create_network_body(self, description):
+        src_body = networks.get_body(
+            self._source_client, self.payload['src_network_id'])
+        dest_network_name = self.get_new_network_name()
+        dest_tenant_id = self.payload['dest_tenant_id']
+        body = {'name': dest_network_name,
+                'tenant_id': dest_tenant_id,
+                'project_id': dest_tenant_id,
+                'description': description}
+
+        for k in src_body:
+            if k not in body and k != 'subnets':
+                body[k] = src_body[k]
+
+        return body
+
+    def execute_operations(self):
+        super(NetworkCreationAction, self).print_operations()
+        done = self.check_already_done()
+        dest_network_name = self.get_new_network_name()
+        if done["done"]:
+            LOG.info(
+                "Network named '%s' already exists.",
+                dest_network_name)
+            return done["result"]
+
+        LOG.info("Creating destination Network with name '%s'" %
+                 dest_network_name)
+
+        description = (self.NEW_NETWORK_DESCRIPTION %
```

Comment:
Please try to keep statements on one line wherever possible:
```python
description = (
    self.NEW_NETWORK_DESCRIPTION % self.get_source_network_name())
```
If that line's over 79 chars, you could also try writing it like this:
```python
description = (self.NEW_NETWORK_DESCRIPTION % (
   self.get_source_network_name()))
```
I know the extra parens seem redundant but seeing "hanging indents" (like on line 1196) drives me nuts, and this extra parens method alleviates that while still being pep8 compliant.